tests/testthat/test-renderer1-axis-angle-rotate.R

```diff
@@ -46,6 +50,7 @@ test_that('axis.text.x=element_text(angle=90) means transform="rotate(-90)"', {
          not=sg)
   info <- animint2HTML(map)
   expect_rotate_anchor(info, "-90", "end")
+  
```

Comment:
please undo all empty line additions

Reply:
are you looking at the files changed tab?

Reply:
@tdhock I will look at files changed tab to remove empty line additions